packages/block-editor/src/components/block-draggable/draggable-chip.js

```diff
@@ -43,7 +44,11 @@ export default function BlockDraggableChip( { clientIds } ) {
 					<FlexItem>
 						<BlockIcon icon={ icon } />
 					</FlexItem>
-					<FlexItem>{ `(${ clientIds.length })` }</FlexItem>
+					{ clientIds.length > 1 && (
+						<FlexItem>
+							{ `${ clientIds.length }` } { __( 'blocks' ) }
```

Comment:
I would appreciate help writing this line in a way that is better for translators. I tried this:

```
						{ sprintf(
								/* translators: %s: number of blocks. */
								_n( '%d blocks', clientIds.length ),
								clientIds.length
							) }
```

however that makes my linter crash with a `TypeError: candidate.match is not a function` error.

Reply:
You could move it higher up to a constant: `const string = _n( '%d blocks', clientIds.length );`

Reply:
I was not able to get this one to work. Or rather, I did get it to work, like so:

```
	/* translators: %d: number of blocks. */
	const blocks = _n( '%d blocks', clientIds.length );

	return (
		<div className="block-editor-block-draggable-chip-wrapper">
			<div className="block-editor-block-draggable-chip">
				<Flex
					justify="center"
					className="block-editor-block-draggable-chip__content"
				>
					<FlexItem>
						<BlockIcon icon={ handle } />
					</FlexItem>
					{ clientIds.length === 1 && (
						<FlexItem>
							<BlockIcon icon={ icon } />
						</FlexItem>
					) }
					{ clientIds.length > 1 && (
						<FlexItem>
							{ sprintf( blocks, clientIds.length ) }
						</FlexItem>
					) }
				</Flex>
			</div>
		</div>
	);
```

but the linter complains that

`28:17  error  Translate function arguments must be string literals  @wordpress/i18n-no-variables`

Any insights?

Reply:
@jasmussen I'll fix and push :)

Reply:
Fixed. In this case, we don't need `_n`, because we don't need a translation for singular, only multiple.